product/product.html

```diff
@@ -0,0 +1,320 @@
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> Gym Shoes
+          </div>
+
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> Running Shoes
+          </div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> Jordan
+          </div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> Cycling Shoes
+          </div>
+          <div class="check-list-item">+ More</div>
+        </div>
+        <div class="list">
+          <div class="list-title">PRICE</div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> $0 - $25
+          </div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> $25 - $100
+          </div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> $100 - $150
+          </div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> $150 - $280
+          </div>
+        </div>
+        <div class="list">
+          <div class="list-title">RATING <i class="bi bi-star-fill"></i></div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> 5 Only
+          </div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> 4 And Up
+          </div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> 3 And UP
+          </div>
+          <div class="check-list-item">
+            <input name="checkbox" type="checkbox" /> 2 And UP
+          </div>
+        </div>
+      </aside>
+      <main class="main-product">
+        <div class="grid-three">
+            <div class="card-component">
+                <div class="card-comp-img">
+                  <img
+                    class="card-top-img"
+                    src="../assets/nike-air-zoom.webp"
+                    alt="loading"
+                    srcset=""
+                  />
+                </div>
+                <div class="card-badge">new</div>
+                <div class="card-comp-contant">
+                  <h2 class="card-h2">Air Jordan 4</h2>
+                  <h3 class="card-h3">by Nike</h3>
+                  <p class="card-para">
+                   <h5>RS 10,000</h5>
+                   <p class="discount-tag">30% off</p>
+                  </p>
+                </div>
+                <div class="card-btn-footer">
+                  <button class="btn btn-outline">Add to Cart</button>
+                </div>
+              </div>
+              <div class="card-component">
+                <div class="card-comp-img">
+                  <img
+                    class="card-top-img"
+                    src="../assets/nike-air-zoom.webp"
+                    alt="loading"
```

Comment:
please use different alt text in every image you can do something like loading-image-1, loading-image-2

Reply:
will change it when shift it to react component